modules/sqlite_helpers.py

```diff
@@ -32,6 +31,18 @@ def maybe_create_table(sqlite_file: str) -> bool:
                     );
                 """
             )
+            cursor.execute(
+                """
+                    CREATE TABLE IF NOT EXISTS weekly_winners (
```

Comment:
idk if this is possible, but what if in sqlite we write to this table when a row is written to snapshots

when the row written to snapshots is the first row for a given week

ask ai about "SQLite trigger"s